sdk_extensions/jaeger_remote_sampler/build.gradle

```diff
@@ -25,7 +25,6 @@ dependencies {
 
     testRuntime "io.grpc:grpc-netty-shaded:${grpcVersion}"
 
-    signature "org.codehaus.mojo.signature:java18:1.0@signature"
     signature "net.sf.androidscents.signature:android-api-level-24:7.0_r2@signature"
```

Comment:
Is it worth trying to put this string in a common place?

Reply:
I think we have issue #1808 for that, let me look at it separately from this PR.